engineyard/ey-cookbooks-stable-v5
# [CC-1087] Elixir

## Description of your patch

- Add availability of Elixir to the stack 
- Sets Erlang version at 18.3
- Sets Elixir version 1.3.4
- Both erlang and elixir have a requested version bump in the works

## Recommended Release Notes

Preliminary work for Elixir support

## Estimated risk

Low

## Components involved

Unreleased Elixir stack

## Description of testing done

Followed QA instructions and elixir and erlang validated by distribution team.

## QA Instructions

Testing will require the elixir feature flag

1. Create an elixir app (can be our demo one).
2. Build environment on stable stack.
3. Ensure chef fails
4. Upgrade to target stack
5. Verify chef run completes successfully
6. App Deploy will fail at this point because of other components still in progress
7. Verify the correct version of Elixir and Erlang are installed via `eix elixir` and `eix erlang` respectively
